Source/WebCore/html/HTMLElement.cpp

```diff
@@ -423,7 +423,7 @@ void HTMLElement::attributeChanged(const QualifiedName& name, const AtomString&
         }
         return;
     case AttributeNames::popoverAttr:
-        if (document().settings().popoverAttributeEnabled() && !document().quirks().shouldDisablePopoverAttributeQuirk())
+        if (document().settings().popoverAttributeEnabled() && name == popoverAttr)
```

Comment:
I'm not sure why `name == popoverAttr` was added here. That's already taken care of by the `case` statement directly above.

Reply:
Oh! I know where it is coming from. Starting the revert to the previous code but forgot to drop the popoverAttr. It was previously just a if. 
Thanks.